### PR TITLE
add extra ASF links

### PR DESCRIPTION
**Pull request by pjfanning**
* privacy
* events

### CODE REVIEW

**Review by nvollmar — APPROVED** [review]
lgtm

File: .gitignore [hunk 1]
```diff
@@ -11,10 +11,13 @@ project/plugins/project/
 .history
 .cache
 .lib/
+.idea/
```
Comment (mdedetrich) [inline .gitignore:14]: Build related files should not be put into a projects `.gitignore`, instead they should be done on a local level using a global gitignore (see https://stackoverflow.com/a/7335487)

**Review by mdedetrich — CHANGES_REQUESTED** [review]
Not sure if its intentional but it appears that you have put IDE specific folders/files into `.gitignore`)

**Review by jrudolph — APPROVED** [review]
lgtm

File: .gitignore (hunk 1 above)
Comment (mdedetrich) [reply .gitignore:14]: I wanted to at one point actually go through all of the projects and remove them and document to use a global git ignore

Comment (pjfanning) [reply .gitignore]: I removed the .gitignore changes

**Review by mdedetrich — APPROVED** [review]
lgtm

**Pull request merged**